pinax/stripe/actions/charges.py

```diff
@@ -61,20 +64,36 @@ def create(amount, customer, source=None, currency="usd", description=None, send
         raise ValueError(
             "You must supply a decimal value representing dollars."
         )
-    stripe_charge = stripe.Charge.create(
+    kwargs = dict(
         amount=utils.convert_amount_for_api(amount, currency),  # find the final amount
         currency=currency,
         source=source,
         customer=customer,
         description=description,
-        capture=capture,
+        capture=capture
+    )
+    if stripe_account:
+        kwargs['destination'] = {
+            'account': stripe_account
+        }
```

Comment:
Should there be support for direct charges, too?
https://stripe.com/docs/connect/charges#choosing-approach

Reply:
@blueyed I can adjust this to allow for both direct and destination charges.

Reply:
I've now done this - take a peek!